README.md

```diff
@@ -13,32 +13,48 @@ An Airflow operator for [Great Expectations](greatexpectations.io), a Python lib
 
 ```
 pip install airflow-provider-great-expectations
 ```
 
 In order to run the `BigQueryOperator`, you will also need to install the relevant dependencies: `pybigquery` and `apache-airflow-providers-google`
 
+Depending on your use-case, you might need to add `ENV AIRFLOW__CORE__ENABLE_XCOM_PICKLING=true` to your Dockerfile to enable XCOM to pass data between tasks.
+
 
 ## Modules
 
-[Great Expectations Operator](./great_expectations_provider/operators/great_expectations.py): A base operator for Great Expectations. Import into your DAG via: 
+[Great Expectations Operator](https://github.com/great-expectations/airflow-provider-great-expectations/blob/main/great_expectations_provider/operators/great_expectations.py): A base operator for Great Expectations. Import into your DAG via: 
 
 ```
 from great_expectations_provider.operators.great_expectations import GreatExpectationsOperator
 ```
 
 
-[Great Expectations BigQuery Operator](./great_expectations_provider/operators/great_expectations_biquery.py): An operator for Great Expectations that provides some pre-set parameters for a BigQuery Datasource and Expectation, Validation, and Data Docs stores in Google Cloud Storage. The operator can also be configured to send email on validation failure. See the docstrings in the class for more configuration options. Import into your DAG via: 
+[Great Expectations BigQuery Operator](https://github.com/great-expectations/airflow-provider-great-expectations/blob/main/great_expectations_provider/operators/great_expectations_bigquery.py): An operator for Great Expectations that provides some pre-set parameters for a BigQuery Datasource and Expectation, Validation, and Data Docs stores in Google Cloud Storage. The operator can also be configured to send email on validation failure. See the docstrings in the class for more configuration options. Import into your DAG via: 
 
 ```
 from great_expectations_provider.operators.great_expectations_bigquery import GreatExpectationsBigQueryOperator
 ```
 
 ## Examples
 
-See the [**examples**](./great_expectations_provider/examples) directory for an example DAG with some sample tasks that demonstrate operator functionality. The example DAG file contains a comment with instructions on how to run the examples.
+See the [**example_dags**](https://github.com/great-expectations/airflow-provider-great-expectations/tree/main/great_expectations_provider/example_dags) directory for an example DAG with some sample tasks that demonstrate operator functionality. The example DAG file contains a comment with instructions on how to run the examples.
+
+Note that to make these operators work, you will need to change the value of [`enable_xcom_pickling`](https://github.com/apache/airflow/blob/master/airflow/config_templates/default_airflow.cfg#L181) to `true` in your airflow.cfg.
+
+To run these examples:
```

Comment:
I'm ok with giving the steps for Astro here, but it should be clear that it's not necessary. I'm thinking either have two paths:
### To run these examples:

**Option 1: Run with a local Airflow installation**
Point your dag_folder at the example_dags directory and run Airflow as usual. (or something like that)

**Option 2: Run with [Astronomer](https://www.astronomer.io)**
Steps below.